### PR TITLE
[VMware] Update vmware api version to 2021-12-01

### PR DESCRIPTION
**Pull request by lijinpei2008**
Add az vmware placement-policy command group
Add az vmware vm command group
Updated az vmware private-cloud create with new Identity parameter
Updated az vmware private-cloud update with new Identity parameter

Design: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1132

### CODE REVIEW

File: src/VMware/custom/README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ These provide functionality to our HTTP pipeline and other useful features. In s
 ### Attributes
 For processing the cmdlets, we've created some additional attributes:
 - `Microsoft.Azure.PowerShell.Cmdlets.VMware.DescriptionAttribute`
-  - Used in C# cmdlets to provide a high-level description of the cmdlet. This is propegated to reference documentation via [help comments](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_comment_based_help) in the exported scripts.
+  - Used in C# cmdlets to provide a high-level description of the cmdlet. This is propegated to reference documentation via [help comments](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help) in the exported scripts.
```
Comment (BethanyZhou) [inline src/VMware/custom/README.md:35]:
```suggestion
  - Used in C# cmdlets to provide a high-level description of the cmdlet. This is propegated to reference documentation via [help comments](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_comment_based_help) in the exported scripts.
```

File: src/VMware/.gitignore [hunk 2]
```diff
@@ -6,7 +6,6 @@ internal
 exports
 tools
 custom/*.psm1
-custom/autogen-model-cmdlets
```
Comment (BethanyZhou) [inline src/VMware/.gitignore:9]: custom/autogen-model-cmdlets should be ignored.

File: src/VMware/examples/Get-AzVMwarePlacementPolicy.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+### Example 1: List
```
Comment (BethanyZhou) [inline src/VMware/examples/Get-AzVMwarePlacementPolicy.md:1]: Please provide a semantic title for example

File: src/VMware/examples/Lock-AzVMwareVirtualMachineMovement.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
+
+{{ Add output here }}
```
Comment (BethanyZhou) [inline src/VMware/examples/Lock-AzVMwareVirtualMachineMovement.md:5]: Please fill placeholder.

**Review by BethanyZhou — CHANGES_REQUESTED** [review]

File: src/VMware/examples/Lock-AzVMwareVirtualMachineMovement.md (hunk 4 above)
Comment (lijinpei2008) [reply src/VMware/examples/Lock-AzVMwareVirtualMachineMovement.md]: The command is invalid and has been hidden

File: src/VMware/examples/Get-AzVMwarePlacementPolicy.md (hunk 3 above)
Comment (lijinpei2008) [reply src/VMware/examples/Get-AzVMwarePlacementPolicy.md]: Done

File: src/VMware/custom/README.md (hunk 1 above)
Comment (lijinpei2008) [reply src/VMware/custom/README.md]: Done

Comment (lijinpei2008) [reply src/VMware/custom/README.md]: The file was generated using the latest version of the autorest command

**Review by BethanyZhou — APPROVED** [review]
LGTM. 
To do list:
[] Move it from generation to master
[] Add change log: 
Upgrade API version to 2021-12-01, added cmdlets `Get-AzVMwareVirtualMachine` and `Get/New/Update/Remove-AzVMwarePlacementPolicy`

**Pull request merged**